Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1213,6 +1225,9 @@
 /* A display format for oversize text messages. */
 "OVERSIZE_TEXT_DISPLAY_FORMAT" = "%@…";
 
+/* Format for percentages, e.g. 65%. Embeds {{percentage}}, e.g. 65. */
+"PERCENTAGE_FORMAT" = "%@%%";
```

Comment:
This string will be a pain to localize since our translators won't understand that %% is an escape sequence for %.  I presume that we'll use some kind of actual progress indicator in the production UI though so I'm not worried about it too much.

Reply:
Alternatively we could get rid of this admittedly confusing translation and use a cached numberFormatter

```
numberFormatter = [[NSNumberFormatter alloc] init];
[numberFormatter setNumberStyle:NSNumberFormatterPercentStyle];
[numberFormatter setMaximumFractionDigits:0];
[numberFormatter setMultiplier:@1];
```

Reply:
Done.